.github/workflows/rust.yml

```diff
@@ -0,0 +1,99 @@
+name: Rust
+
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+
+env:
+  CARGO_TERM_COLOR: always
+  CI: 1
+
+jobs:
+  cargo-audit:
+    name: Audit Rust vulnerabilities
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: rust-audit-check
+      uses: actions-rs/audit-check@v1.2.0
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+
+  cargo-test:
+    name: Cargo Test
+    runs-on: ubuntu-latest
+
+    container:
+      image: archlinux:base-devel
+      options: --privileged
```

Comment:
What does this do and why is it needed?

Reply:
Needed for using archlinux instead of ubuntu (default linux container)

Reply:
Works for me without it at https://github.com/Morganamilo/alpm.rs/blob/master/.github/workflows/alpm.yml

Reply:
oh, I apologize for not immediately thinking to do so :(